feasibility-dsf-process-docker-test-setup/README.md

```diff
@@ -57,21 +61,22 @@ your `/etc/hosts`. The next line in the command is about Client and CA Certifica
 After that, you can start the ZARS Business Process Engine in a separate terminal:
 
 ```sh
-docker-compose up zars-bpe-app
+docker-compose up -d zars-bpe-app && docker-compose logs -f zars-fhir-app zars-bpe-app
```

Comment:
I normally use only one terminal window for logging the output of a DSF instance. In my opinion, this makes it easier to follow the actual output of a process execution since FHIR and BPE output follow sequentially in one window.

Should I change it back to the previous version or should I add a note to the Readme instead?

Reply:
`After that, you can start the ZARS Business Process Engine in a separate terminal:` is the crucial part within the Readme as it explicitly mentiones a separate terminal.

Your solution is not inferior, but we should make this clear within the Readme. The outcome is equal since all BPE services depend on their associate FHIR service (according to your changes) and thus implicitly triggering its start.

So yeah, we can streamline this by using your suggested setup and adjust the Readme accordingly. I believe, anyone developing with `docker-compose` will be able to adjust the setup to fit the requirements at hand anyway.